eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -17,12 +17,15 @@ parameters:
   - name: PreTestSteps
     type: object
     default: []
+  - name: BuildOptions
+    type: string
+    default: '$(DefaultOptions) -T 1C -DskipTests -Dgpg.skip -DtrimStackTrace=false -Dmaven.javadoc.skip=true -Dspotbugs.skip=true -Dcheckstyle.skip=true -Drevapi.skip=true -Djacoco.skip=true'
   - name: TestOptions
     type: string
     default: '$(DefaultOptions) -DtrimStackTrace=false -Dmaven.javadoc.skip=true -Dspotbugs.skip=true -Dcheckstyle.skip=true -Drevapi.skip=true -Dparallel-test-playback'
   - name: TestGoals
     type: string
-    default: verify
+    default: 'surefire:test'
```

Comment:
I would delete this whole parameter block, and replace all references to `${{ parameters.TestGoals }}` to `$(TestGoals)`, and make sure you add `"TestGoals": "verify"` to the base test matrix.